File: 문민우/프로그래머스/Level1/두개뽑아서더하기.java
package Level1;

import java.util.Set;
import java.util.TreeSet;

class 두개뽑아서더하기 {
    public int[] solution(int[] numbers) {

        Set<Integer> result = new TreeSet<>();

        for(int i = 0; i < numbers.length; i++) {
            for(int j = i + 1; j < numbers.length; j++) {
                result.add(numbers[i] + numbers[j]);
            }
        }

        return result.stream().mapToInt(Integer::intValue).toArray();
    }
}

File: 이솔/프로그래머스/Level1/두 개뽑아서 더하기.py
def solution(numbers):
    a = len(numbers)
    answer = []
    for i in range(0, a-1):
        for j in range(i+1, a):
            k = numbers[i]+numbers[j]
            if k not in answer:
                answer.append(k)
    answer.sort()
    print(answer)

    return answer
